### ISSUE TITLE
Separate C Conformance page

### ISSUE THREAD

**Issue by h-vetinari** [MicrosoftDocs/cpp-docs#2599]
Even though #407 has been closed, there is still no explicit mention of C conformance, much less a table for tracking the various different aspects. Searching for "C Conformance" leads to the [C++ Conformance page](https://docs.microsoft.com/en-us/cpp/overview/visual-cpp-language-conformance), which has a tiny snippet:

<details>

> **C11** Compiler support for C11 requires Visual Studio version 16.8, or higher. C11 library support requires Windows SDK version 20211, or higher. The Universal CRT implements the parts of the C11 Standard Library that are required by C++17, with the exception of C99 `strftime()` E/O alternative conversion specifiers, and C11 `aligned_alloc()`. The latter is unlikely to be implemented because the Windows operating system doesn't support aligned allocations.

</details>

There _is_ an [install](https://docs.microsoft.com/en-us/cpp/overview/install-c17-support) page for C11/C17, but that doesn't help with conformance.

Having an explicit conformance page would be highly appreciated, since Visual Studio 16.8 has so far [not implemented](https://devblogs.microsoft.com/cppblog/c11-and-c17-standard-support-arriving-in-msvc/#whats-not) any of the C11 [optionals](https://en.wikipedia.org/wiki/C11_(C_standard_revision)#Optional_features) (though it mentions that atomics & threading are on the roadmap) - those differences are crucial for authors having to work with the lowest common denominator across several OSes.

This will also become ever more relevant with the [growing](https://en.wikipedia.org/wiki/C2x) [list](http://www.open-std.org/jtc1/sc22/wg14/www/docs/n2583.pdf) of features for C2x/C23.

**Comment by h-vetinari** [comment]
There's now a [section](https://docs.microsoft.com/en-us/cpp/overview/visual-cpp-language-conformance?view=msvc-170#c-standard-library-features-1) regarding "C Standard library features". However, the language side seems to be missing.

In particular, while C++ conformance tracks the as-yet unreleased C++23, there's no sign of conformance for the upcoming C23 - this is going to be a very important update to the C standard (with lots of substantial improvements), so I sincerely hope that MSFT will not things slide again like for C99.  As a reference, here's the conformance page for [clang](https://clang.llvm.org/c_status.html), which contains a relatively up-to-date list of papers that have been accepted into C23 so far.

Note that C23 was supposed to be effectively feature-complete already, so while some papers are still in flight, there shouldn't be huge changes anymore - certainly stable enough to reflect on a conformance page.

**Comment by h-vetinari** [comment]
@TylerMSFT @corob-msft 
Any update here? 17.5 now has C11 atomics support (even if experimantal), so this information should be present somewhere. Also, C23 is now feature-complete and just going through the ISO-motions for publications. A _lot_ of people (especially on the C++ side) are waiting for features like `#embed`, which happened to land in the C standard before they did in C++. This really should be visible somewhere.

CC @TartanLlama @StephanTLavavej @CaseyCarter

**Comment by TylerMSFT** [comment]
Agreed. However, as I'm the only one maintaining over 10,000 C++ topics now, as well as writing all the new ones, it will be a while until I can get to this.

**Comment by h-vetinari** [comment]
According to the 17.8 preview 2 [blogpost](https://devblogs.microsoft.com/visualstudio/visual-studio-2022-17-8-preview-2-has-arrived/#c11-threading-support), C11 threads are now supported! 🥳 

It looks like the C conformance [section](https://learn.microsoft.com/en-us/cpp/overview/visual-cpp-language-conformance?view=msvc-170#c-standard-library-features-1) gained a bit more detail too, thanks for that! Still, that table could probably use an update for the two topics above (much less for C23). 🙃
